Move Flaky CircleCI E2E Tests to TravisCI

U8NWXD opened the pull request:
# Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of n/a.
2. This PR does the following:

Tests on Travis can be rerun individually, so they are easier to handle
when they flake. This commit moves the following flaky tests to
TravisCI:

* Embedding

## Essential Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## PR Pointers

- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays
- Never force push. If you do, your PR will be closed.

## Test Run History

1. Embedding suite passed: https://travis-ci.com/github/oppia/oppia/builds/176223472

![Screen Shot 2020-07-19 at 20 01 25](https://user-images.githubusercontent.com/19878639/87888401-b857f300-c9fa-11ea-8cc4-b2d623441d47.png)

2. Embedding suite passed

![Screen Shot 2020-07-19 at 20 33 43](https://user-images.githubusercontent.com/19878639/87889359-2ef6ef80-c9ff-11ea-973c-cafa09b29c46.png)

3. Embedding suite passed

![Screen Shot 2020-07-19 at 20 55 12](https://user-images.githubusercontent.com/19878639/87889946-2c49c980-ca02-11ea-9aae-f7af9e35d5fe.png)

4. Embedding suite passed

![Screen Shot 2020-07-19 at 21 56 17](https://user-images.githubusercontent.com/19878639/87892290-b9911c00-ca0a-11ea-8ede-8ea603a3f0c9.png)

5. Embedding suite passed

![Screen Shot 2020-07-19 at 22 13 17](https://user-images.githubusercontent.com/19878639/87893172-14c40e00-ca0d-11ea-8eb8-97bacafc3ccc.png)

U8NWXD committed:
Move Flaky CircleCI E2E Tests to TravisCI

Tests on Travis can be rerun individually, so they are easier to handle
when they flake. This commit moves the following flaky tests to
TravisCI:

* Embedding
* Subscription
* Profile Features

U8NWXD committed:
Update E2E CI Test Script

iamprayush commented:
@U8NWXD Please note the CI failures, thanks!

Showtim3 reviewed (approved):
LGTM for codeowners

nithusha21 reviewed (approved):
LGTM, thanks!

Showtim3 commented:
Hey @U8NWXD Mind the merge conflicts.

iamprayush commented:
Hey @U8NWXD, Please note the merge conflicts, thanks!

U8NWXD committed:
Merge branch 'develop' into move-e2e-tests

U8NWXD commented:
@vinitamurthi @BenHenning @DubeySandeep I've fixed the merge conflicts. PTAL as codeowners

BenHenning commented:
Thanks @U8NWXD--this generally LGTM, but do you have a sample run of these running successfully on Travis that can be linked here in the PR?

U8NWXD commented:
@BenHenning I've added a screenshot and link to the top comment

DubeySandeep commented on .circleci/config.yml:
```diff
@@ -215,10 +207,6 @@ jobs:
-      - run:
-          name: Run e2e profileFeatures test
-          command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="profileFeatures" --prod_env
```
If we are planning to move this test back to circleCi then I and Kevin have created PR to fix issues in this suite so maybe we can leave it to cicleCI? (Leaving it to you to decide I'm fine with either!)

BenHenning replied:
@DubeySandeep can you link to the PR that fixes the issues in this suite for reference? If the underlying issues are fixed, do we have any other advantages to moving to TravisCI @U8NWXD?

U8NWXD replied:
Let's leave the subscriptions and profileFeatures tests in CircleCI so we can better evaluate whether they're fixed

BenHenning commented:
Apologies for the delay, but I will need to take a pass on this PR tomorrow. I ran out of time to do so today.

BenHenning commented:
See my comment above--moving this to TravisCI SGTM, but do we still need to do this if the flaky issues were fixed per @DubeySandeep?

BenHenning reviewed (approved):
If there are advantages to moving to TravisCI despite the potential fixes, then this LGTM.

U8NWXD committed:
Leave Subscriptions, ProfileFeatures on CircleCI

Recent fixes have reduced the flakiness of these suites, so we are
leaving them on CircleCI
